src/lib/pages/backoffice/EItem/EItemSearch/EItemListEntry.js

```diff
@@ -28,9 +27,10 @@ export default class EItemListEntry extends Component {
             <Grid.Column computer={6} largeScreen={5}>
               <Item.Meta className="document-authors">
                 <DocumentAuthors
-                  metadata={eitem.metadata.document}
+                  authors={eitem.metadata.document.authors}
+                  hasOtherAuthors={eitem.metadata.document.other_authors}
                   prefix="by "
-                  authorsLimit={10}
+                  limit={10}
```

Comment:
I think this should be the default value for authors limit, and we should pass `5` in the one case is needed.

Reply:
There is already a default limit to `5`